platform/nativescript/runtime/components/list-view.js

```diff
@@ -13,6 +20,9 @@ export default {
         templateSelector: {
             type: Function,
             default: () => 'default'
+        },
+        separatorColor: {
+            type: String
```

Comment:
I tested with and without the property specified. Without it falls back to the NativeScript default, when set it uses the value specified.